core/keyserver/keyserver.go

```diff
@@ -54,8 +54,7 @@ type Server struct {
 func New(tlog tpb.TrillianLogClient,
 	tmap tpb.TrillianMapClient,
 	tadmin tpb.TrillianAdminClient,
-	mutator mutator.Mutator,
-	auth authentication.Authenticator,
+	mutator mutator.Func, auth authentication.Authenticator,
```

Comment:
did you mean to put these on the same line?  split if not.

Reply:
woops.  Thanks for the catch.